src/components/va-back-to-top/va-back-to-top.tsx

```diff
@@ -0,0 +1,78 @@
+import { Component, Element, Host, State, h } from '@stencil/core';
+import classnames from 'classnames';
+
+import 'intersection-observer';
+
+@Component({
+  tag: 'va-back-to-top',
+  styleUrl: 'va-back-to-top.css',
+  shadow: true,
+})
+export class VaBackToTop {
+  revealPixel!: HTMLSpanElement;
```

Comment:
Remind me again what the `!` is for? Non-nullable property?

Reply:
Yup, that's exactly it.

https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#non-null-assertion-operator-postfix-